src/components/CircularProgressBar/CircularProgressBar.protractor.driver.ts

```diff
@@ -1 +1,15 @@
-export {circularProgressBarDriverFactory, CircularProgressBarDriver} from 'wix-ui-core/dist/src/components/CircularProgressBar/CircularProgressBar.protractor.driver';
\ No newline at end of file
+import {$, ElementFinder} from 'protractor';
+import {DriverFactory} from 'wix-ui-core/dist/src/common/BaseDriver.protractor';
+export {circularProgressBarDriverFactory as coreCircularProgressBarDriverFactory, CircularProgressBarDriver as CoreCircularProgressBarDriver} from 'wix-ui-core/dist/src/components/CircularProgressBar/CircularProgressBar.protractor.driver';
+
+export interface CircularProgressBarDriver extends CoreCircularProgressBarDriver {
+  /** Get tooltip that appears on error icons */
+  getTooltip?: () => ElementFinder;
+}
+
+export const circularProgressBarDriverFactory: DriverFactory<CircularProgressBarDriver> = (element: ElementFinder): CircularProgressBarDriver => {
+  return {
+    ...coreCircularProgressBarDriverFactory($(`[data-hook='circular-progress-bar']`)),
```

Comment:
I am guessing that instead of `$(`[data-hook='circular-progress-bar']`` it should be `element`